fennel/sources/sources.py

```diff
@@ -491,18 +489,22 @@ def __init__(
 
     def _validate(self) -> List[Exception]:
         exceptions: List[Exception] = []
-        if self.format not in ["csv", "json", "parquet", "hudi"]:
+        if self.format not in ["csv", "json", "parquet", "hudi", "delta_lake"]:
```

Comment:
we can call it just `delta` (even their website is https://delta.io/) instead of `delta_lake`